source/fundamentals/auth.txt

```diff
@@ -329,17 +329,13 @@ mechanism:
 The ``X.509`` authentication mechanism uses
 :wikipedia:`TLS <Transport_Layer_Security>` with X.509 certificates to
 authenticate your user, identified by the relative distinguished names
-(RDNs) of your client certificate.
-
-To retrieve the X.509 certificate *subject* which is used as your
-*username* for authentication, see the MongoDB server :manual:`X.509
-tutorial </tutorial/configure-x509-client-authentication/#add-x-509-certificate-subject-as-a-user>`
-for more information.
+(RDNs) of your client certificate. When you specify the ``X.509``
+authentication mechanism, the server automatically retrieves your client
```

Comment:
Not sure what this is supposed to mean.  Whether or not you use this mechanism, the client will send its certificate to the server.  When you specify this mechanism, the server will also authenticate the connection based on the name that's in the certificate

Reply:
Whoops. Quick question: would it be appropriate to mention they need to have their trust store configured for this?

Reply:
No.  The trust store is about verification of the server's certificate.